app/models/cart.rb

```diff
@@ -1,10 +1,11 @@
 class Cart
   include ActionView::Helpers::NumberHelper
 
-  attr_reader :contents
+  attr_reader :contents, :quantity
 
   def initialize(initial_contents)
     @contents = initial_contents || {}
+    @quantity = contents.values.sum
```

Comment:
Instead of having an instance variable for quantity, maybe we could instead have a virtual attribute? Something like:

```ruby
  def quantity
    contents.values.sum
  end
```

This should allow us to get rid of the instance variable along with the `attr_reader`